src/CMakeLists.txt

```diff
@@ -157,3 +157,19 @@ automaton_test(storage blobstore_test)
 automaton_test(storage persistent_blobstore_test)
 # automaton_test(storage persistent_storage_test)
 automaton_test(storage persistent_vector_test)
+
+add_library(
+  blockchain_cpp_node
+  automaton/examples/node/blockchain_cpp_node/blockchain_cpp_node.cc
+)
+
```

Comment:
I would add target_link_libraries for blockchain_cpp_node to automaton-core as well.